### PR TITLE
Re-adds TRAIT_TRASHMAN to the Hydraulic arms module

### PR DESCRIPTION
**Pull request by Syrox25**
## About The Pull Request
Mirror PR #19959 seems to have removed this feature and I dont believe it was intentional, Cant find any documentation of it.

## How This Contributes To The Skyrat Roleplay Experience
Restoration, Gives Cargo modsuit its disposal functionality back.

## Proof of Testing
Deployed modsuit, took no damage.

<details>
<summary>Screenshots/Videos</summary>
  

https://github.com/Skyrat-SS13/Skyrat-tg/assets/75050485/fbc35c08-abc0-4ad4-838d-92afa13ae6c0

image

</details>

## Changelog

:cl:
fix: Cargo loaders are once again disposal and dishwasher safe
/:cl:

### CODE REVIEW

**Comment by Syrox25** [comment]
been moved to `modular_skyrat/code`
I think this is correct but please double check :(

Retested and same results as inital video

**Comment by vinylspiders** [comment]
Good catch--yeah that seems likely a merge error. Now that it's modular it will be less likely to be lost in such situations, thanks for doing that!

File: modular_skyrat/master_files/code/modules/mod/modules/modules_supply.dm [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/obj/item/mod/module/hydraulic/on_suit_activation()
+		ADD_TRAIT(mod.wearer, TRAIT_TRASHMAN, MOD_TRAIT)
```
Comment (vinylspiders) [inline modular_skyrat/master_files/code/modules/mod/modules/modules_supply.dm:3]:
```suggestion
		. = ..()
		ADD_TRAIT(mod.wearer, TRAIT_TRASHMAN, MOD_TRAIT)
```
Only thing is I would just add this, so if upstream ever adds their own implementation of the proc we call that as well. It doesn't right now for the hydraulic module but this future proofs it.

File: modular_skyrat/master_files/code/modules/mod/modules/modules_supply.dm [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+/obj/item/mod/module/hydraulic/on_suit_deactivation(deleting = FALSE)
+		REMOVE_TRAIT(mod.wearer, TRAIT_TRASHMAN, MOD_TRAIT)
```
Comment (vinylspiders) [inline modular_skyrat/master_files/code/modules/mod/modules/modules_supply.dm:7]:
```suggestion
		. = ..()
		REMOVE_TRAIT(mod.wearer, TRAIT_TRASHMAN, MOD_TRAIT)
```

**Review by vinylspiders — CHANGES_REQUESTED** [review]

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**